pkg/result/processors/path_prefixer.go

```diff
@@ -0,0 +1,37 @@
+package processors
+
+import (
+	"path"
+
+	"github.com/golangci/golangci-lint/pkg/result"
+)
+
+// PathPrefixer adds a customizable prefix to every output path
+type PathPrefixer struct {
+	prefix string
+}
+
+var _ Processor = new(PathPrefixer)
+
+// NewPathPrefixer returns a new path prefixer for the provided string
+func NewPathPrefixer(prefix string) *PathPrefixer {
+	return &PathPrefixer{prefix: prefix}
+}
+
+// Name returns the name of this processor
+func (*PathPrefixer) Name() string {
+	return "path_prefixer"
+}
+
+// Process adds the prefix to each path
+func (p *PathPrefixer) Process(issues []result.Issue) ([]result.Issue, error) {
```

Comment:
can you add the test for this method? Shout out if you need any help.

Reply:
Added a test